docs/reference/notices.md

```diff
@@ -0,0 +1,18 @@
+---
+id: notices
+title: "Notices"
```

Comment:
Should we call this page "Security Notices", otherwise I'm not sure I would find where to report a security vulnerability. Or is "Notices" imply security and I'm just not aware of it?

Reply:
This is the title in the sidebar on docs.camunda.org, but looking closer I realize contextually it sits in the "Security" section. I will adjust the name.